README.md

```diff
@@ -54,15 +54,15 @@ https://hosted.weblate.org/projects/redreader/strings/
 Building
 --------
 
-RedReader is built using Gradle. On Linux, simply run:
+RedReader is built using Gradle. On Linux, run:
 
     ./gradlew installDebug
 
 
 License
 -------
 
-RedReader is licensed under the GPL, version 3. A copy of the license is
+RedReader is licensed under the GPL version 3 or any later version. A copy of the license is
```

Comment:
I can't retroactively relicense the app without getting permission from contributors ;)

Reply:
Thought it already was. Not sure what the effective licensing is if it is neither specifically "only", and doesn't have "or any later version" in each file.